### DISCUSSION TITLE
Monthly Model Mural

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#101] [people; relating]
To extend our discussion in #7, I am logging models I wish to revisit for coloring. Few tips:
- time (red) and frac (blue) are interchangable
- unit conversion (defect per task) is underlined
- observed time series vector (plum) is generated by convoluting latent time series vector from population distribution (time series vector) and observational distribution (e.g. neg.binom). I think words (local-global, individual-population) binarize parameter concepts, and is obstacle for multi(>2) level hierarchy. Same with prior and likelihood distribution as prior distribution can be the likleihood distribution for the layer underneath. Same with stock and flow (as speed is stock of acceleration). But would we have words to replace this?

![image](https://user-images.githubusercontent.com/30194633/203575790-72161ff7-0953-4260-8088-d0fd840ac52b.png)

**Comment by tomfid** [comment]
Part of this will be available in Vensim fairly soon, styling variables by type according to a user-defined skin. We probably won't have the ability to distinguish a unit converter from a time constant though.

**Reply by hyunjimoon** [reply]
https://github.com/Data4DM/BayesSD/discussions/7#discussioncomment-4219864 was my first draft of Color Semantic and Syntax. Could you please introduce your system in the model below and may we sync colors for higher communication efficiency? :-)

**Comment by hyunjimoon** [comment]
![image](https://user-images.githubusercontent.com/30194633/203638669-45481419-1b14-42cd-bbe7-72c076419712.png)

**Comment by tomfid** [comment]
The purpose of color here is to highlight features (loops) of interest, so I can refer to them in the accompanying text. Sometimes I identify the color with the meaning of the loop (i.e. red = bad = vicious cycle) but that's often not possible because the loop can run either direction.

**Reply by hyunjimoon** [reply]
Thanks, for loops, I have been using 
- red as reinforcing (red = time = non-negative = single direction)
- green as balancing (green = space = can have negative = bi-direction)
- orange as operationalizing (actionable) loop or flow

I haven't tried loop-coloring in vensim in the fear of it getting mixed with parameter-coloring. However, let
s recall Table function parameter zips causal loop from #38. Dynamic zip and unzip (like cell division) would be the end-goal, but meanwhile, consistent use of color for loop and parameter seems helpful :)
